src/Dict.cc

```diff
@@ -530,80 +328,82 @@ TEST_CASE("dict robust iteration replacement")
 	delete val4;
 	}
 
+// TODO: should this test be rewritten for the new iteration style?
+/*
 TEST_CASE("dict iterator invalidation")
-	{
-	PDict<uint32_t> dict;
+    {
+    PDict<uint32_t> dict;
```

Comment:
These and the following changes all look extremely weird - I know that this is now commented out, but it simultaneously changes all indentation tabs to spaces.

That is not right, is it?

Reply:
I went back and reimplemented that test using the new iterators, like I should have done in the first place.